lib/cocoapods/installer/analyzer.rb

```diff
@@ -909,58 +893,6 @@ def generate_sandbox_state
 
       #-----------------------------------------------------------------------#
 
-      # @!group Analysis internal products
```

Comment:
all of this was moved, public methods at the top, all other methods are private now.